lib/ui/home/near_parkings.dart

```diff
@@ -211,12 +235,12 @@ class ListOfParkingLots extends StatelessWidget {
             name: parking.name!,
             numberSpots: parking.carSpotsAvailable!.toString(),
             price: parking.price!.toString(),
-            // TODO: Get current location for real
             distance: Haversine.haversine(
-                    0.0,
```

Comment:
Depending on the device and whether location permissions are granted, it may generate null pointer errors. I would recommend initializing the object with fixed numbers and then updating the value.

Reply:
Good catch.

In dart, final variables get their value when declared and can't be changed. I made an alternative in b1ed4f9